### ISSUE TITLE
Give program admins the ability to download their program information

### ISSUE THREAD

**Issue by lindsayzad** [seattle-uat/civiform#1590]
**Is your feature request related to a problem? Please describe.**
Currently, program admins have to keep two separate copies of their program information; one on civiform to be published, and one internally that keeps track of the text, question type, and other metadata to be easily shared internally with other government staff.  One program admin does this in a separate word document that lists every question and all the metadata.  That word doc is shared with other government staff to check the question information for correctness, any suggestions, and to extract the text for translations.  When an edit needs to be made, the program admin has to make the edit in both places, and keep track of the two to make sure they don't diverge.

**Describe the solution you'd like**
It would be much easier and more usable for the program admin if they could export all of their program information to a text file, or similar format, instead of having to keep track of all of it themselves.  This will save a lot of their time, and also make the process less error prone.  It should be in a format that's human readable, clearly organized and formatted, and can be shared easily.  A csv or pdf may also make sense here.

**Additional context**
This is currently a huge headache for program admins.  Even if the program admin isn't getting feedback on it from other staff that don't have accounts on civiform, a process like this will have to happen for every single program in order to extract the text and request translations for it.

**Done when**
There is an easily extractable and readable copy of the program content for the admin to extract and review.
(PM?): Decide which data to include.  All question and program metadata as default
BE: Allow this data to be extracted and formatted correctly in a new file
UX: Design where this export info button will be (in home page? Within the program itself?)
FE: Add the extract button and connect with BE

**Comment by bion** [comment]
@elisekalstad is this still a priority for Seattle?

**Comment by elisekalstad** [comment]
No, this is more of a "nice to have". It's actually working pretty well to have program staff review the questions and text in a Word document.

**Comment by bion** [comment]
@elisekalstad alright if I close it for now then?
